doc/LETS_ENCRYPT_CONFIG.md

```diff
@@ -0,0 +1,49 @@
+Here are the steps to get Let's Encrypt SSL certificates on Chef server
+
+##Prerequisites
+
+1. Chef server must be reachable on port 80 or 443 on a public IP address
```

Comment:
Not necessarily, to get proper validation via HTTPS and Certbot (most popular utility) Let's Encrypt servers would still reach Chef nginx via HTTPS, but the challenge request we would have to be passed to certbot standalone web server, running on another port. 
There is a Certbot nginx plugin but still in alpha, I wouldn't go there.